docs/schema/workflowinfo-element-workflowinfo.md

```diff
@@ -32,9 +33,6 @@ The following sections describe attributes, child elements, and parent elements.
 
 ### Attributes
 
-<table>
-<colgroup>
-<col width="20%" />
 <col width="80%" />
```

Comment:
Not sure what happened here, but let's add back the HTML lines back for the table.